docs/Reference/APIs/PermissioningAPIs.md

```diff
@@ -1317,6 +1317,90 @@ Once majority approvals from network admin accounts is received, the node is mar
         "Action completed successfully"
         ```
 
+### `quorumPermission_transactionAllowed`
+
+This api checks if the account initiating the transction has sufficient permissions to execute the transaction
+
+#### Parameters
+
+* `txArgs`: transaction arguments object
+
+#### Returns
+
+* `status`: `bool` indicating if transaction is allowed or not
+
+!!! example
+    === "JSON-RPC"
+
+        Request:
+
+        ```bash
+        curl -X POST http://127.0.0.1:22000 --data '{"jsonrpc":"2.0","method":"quorumPermission_transactionAllowed", "params":[ {"from":"0xf2cd20ed7904c103ce2ca0ef73fb77539930c59f"}],"id":50}' --header "Content-Type: application/json"
+        ```
+
+        Response:
+
+        ```json
+        {"jsonrpc":"2.0","id":50,"result":true}
+        ```
+
+    === "Geth console"
+
+        Request:
+
+        ```javascript
+        quorumPermission.transactionAllowed({from: eth.accounts[0]})
+        ```
+
+        Response:
+
+        ```json
+        "true"
+        ```
+
+### `quorumPermission_connectionAllowed`
+
+This api checks if the given enode is permissioned to join network
```

Comment:
```suggestion
This api checks if the given enode is allowed to join network
```
Providing a synonym in the description helps to understand what the method does without using the same verb like what you already did on line 1373. Note that permission only appear as a noun in most dictionaries.